src/config.py

```diff
@@ -188,42 +236,66 @@ def from_network(network: Network) -> PaymentConfig:
 
         docs_url = "https://www.notion.so/cownation/Solver-Payouts-3dfee64eb3d449ed8157a652cc817a8c"
 
-        network_short_name = {
-            Network.MAINNET: "eth",
-            Network.GNOSIS: "gno",
-        }
-
         match network:
             case Network.MAINNET:
+                payment_network = EthereumNetwork.MAINNET
                 payment_safe_address = Web3.to_checksum_address(
                     os.environ.get(
-                        "SAFE_ADDRESS", "0xA03be496e67Ec29bC62F01a428683D7F9c204930"
+                        "SAFE_ADDRESS_MAINNET",
```

Comment:
Just realized that the name is not very descriptive. Something like `PAYOUTS_SAFE` would be more descriptive

Reply:
I changed it to `PAYOUTS_SAFE_ADDRESS_${NETWORK}`.